kbase-extension/scss/partials/_bulkImportCell.scss

```diff
@@ -119,8 +119,19 @@ $tab-widget-padding: 4px;
 
         // .kb-bulk-import__filetype_panel__filetype_icon
         &_icon {
-          color: $filetype-icon-color;
           padding: 2px 8px 0;
+          width: 3rem;
+          text-align: center;
+
+          // .kb-bulk-import__filetype_panel__filetype_icon_complete
+          &_complete {
```

Comment:
For state modifiers, I use `--` as the separator, i.e. `.kb-bulk-import__filetype_panel__filetype_icon--complete` and `.kb-bulk-import__filetype_panel__filetype_icon--incomplete`

Reply:
Ok, fixed.